src/main.rs

```diff
@@ -27,12 +27,12 @@ const BILLION: i64 = 1_000_000_000;
 const MILLION: i64 = 1_000_000;
 const THOUSAND: i64 = 1_000;
 
-const BLANKS: &'static str = "Blank lines";
+const BLANKS: &'static str = "blank lines";
 const BLUE: &'static str = "#007ec6";
-const CODE: &'static str = "LoC";
-const COMMENTS: &'static str = "Comments";
-const FILES: &'static str = "Files";
-const LINES: &'static str = "Total lines";
+const CODE: &'static str = "code lines";
```

Comment:
I'd prefer if this was `lines of code`, I know it's different from the rest, but `code lines` doesn't sound correct to me.

Reply:
OK!